src/mock/ray/gcs/gcs_server/gcs_placement_group_scheduler.h

```diff
@@ -67,101 +65,6 @@ class MockGcsPlacementGroupSchedulerInterface
 namespace ray {
 namespace gcs {
 
-class MockScheduleContext : public ScheduleContext {
```

Comment:
To be clear:
All placement group related strategies are moved out from `gcs_placement_group_scheduler.h/.cc` to `bundle_scheduling_policy.h/.cc`, so these mocked strategies could be removed directly.